### ISSUE TITLE
Error: Custom error content image not marked up as presentational (1.1.1)

### ISSUE THREAD

**Issue by Blackbaud-MattGregg** [blackbaud/skyux1#843]
### Expected behavior

Image displayed with custom error content has role="presentation". 

_WAI-ARIA introduces the presentation role as a general device for marking elements as presentational. The presentation role overrides the element's native implicit role, and informs the user agent to not publish the element to the accessibility API. In fact, the preferred way to mark <img> elements as decorative is to use a role="presentation" attribute instead of (or in addition to) an empty alt attribute._ 
https://www.w3.org/TR/wai-aria-practices/#presentation_intro
### Actual behavior

Image element missing alt attribute. Pre-defined error types use background image so don't have this issue. 
http://skyux.developer.blackbaud.com/components/error/

**Comment by Blackbaud-PatrickOFriel** [comment]
In [v1.10.1](https://github.com/blackbaud/skyux/releases/tag/v1.10.1)
